### PR TITLE
Pethats.

### PR DESCRIPTION
**Pull request by kit-katz**
## About The Pull Request
Introducing PET HATS. QUITE A LOT OF EM
also a few pets can now be picked up

adds holding and or head sprites for:

* almost all corgi variants + a pug
* the 2 corgi puppies (one with new handsprites)
* ALL cat variants
* poly the slipery (if you manage to grab him)
* captain's fox for there is only 1
* mice and hamsters
* jani's pet lizard
* chicks(the OTHER kind)
* Crabs
* SNAKES!!
all coded to be on your head... obviously
code ported in #2479 and the code was made usable by imsxz.

imageimageimageimageimageimageimageimageimageAnimationimage

imageimageimageimageimageimageimageimageimageimage
and plenty more (mostly corgi and cat resprites) ^

i take no criticism for anything. the sprites exausted me. if you think you can do better be my guest

## Why It's Good For The Game
have you ever missed your pet that is just locked in your room. or just noticed that all the pets other than ian are barely seen by human and not so human eyes. OR just stuck in your backpack
INTRODUCING HAT PETS. the luster of your favorite pet now with you for all to see

also given the oportunity to give traitors the ability to throw angry snakes at their target over just bombing medical for the 1000th time YES ILL TAKE SAID OPORTUNITY.

the framework was there i just took an oportunity. i doubt cosmetic changes would be a problem. 
unless there is some pet meta im not aware of

## Changelog
:cl:
add: plenty of pets and critters can now be picked up and worn as a hat (ethically)
add: most corgi types, pugs, corgi puppies, Walter, ALL cat types(yes even cak), 
captain's fox for there is only 1, jani's pet lizard,Poly
add: mice, hamsters, chickens and chicks(The other type) crabs and SNAKES!!
imageadd: sprites for the above
/:cl:

### CODE REVIEW

**Comment by Penwin0** [comment]
Merge it before the merge conflicts come

**Comment by Archanial** [comment]
> not that i made the code. and if i knew who it was i would give them credit... THANKS

they were ported in #2479 and the code was made usable by imsxz.

**Comment by kit-katz** [comment]
noted and updated. that lad is a legend

File: code/modules/mob/living/simple_animal/friendly/snake.dm [hunk 1]
```diff
@@ -32,9 +32,21 @@
 	gold_core_spawnable = FRIENDLY_SPAWN
 	obj_damage = 0
 	environment_smash = ENVIRONMENT_SMASH_NONE
+<<<<<<< refs/remotes/BeeStation/master
+<<<<<<< refs/remotes/BeeStation/master
```
Comment (Archanial) [inline code/modules/mob/living/simple_animal/friendly/snake.dm]:
```suggestion
```

File: code/modules/mob/living/simple_animal/friendly/snake.dm [hunk 2]
```diff
@@ -32,9 +32,21 @@
...
 	chat_color = "#26F55A"
+=======
+	mobsay_color = "#26F55A"
+>>>>>>> update
+=======
+	chat_color = "#26F55A"
+>>>>>>> update
```
Comment (Archanial) [inline code/modules/mob/living/simple_animal/friendly/snake.dm]:
```suggestion
	chat_color = "#26F55A"
```

Comment (kit-katz) [reply code/modules/mob/living/simple_animal/friendly/snake.dm]: dunno what you are talking about... that was never there
i totally did not miss it

Comment (ItsTisme) [reply code/modules/mob/living/simple_animal/friendly/snake.dm]: Left overs from the merge conflicts

**Review by utilbird — APPROVED** [review]
Cute!

Though, in future try not to PR from your master branch

**Pull request merged**